src/objects/Mesh.js

```diff
@@ -208,11 +208,43 @@ THREE.Mesh.prototype = Object.assign( Object.create( THREE.Object3D.prototype ),
 				var a, b, c;
 				var index = geometry.index;
 				var attributes = geometry.attributes;
-				var positions = attributes.position.array;
+				var positions;
+				if ( attributes.position instanceof THREE.InterleavedBufferAttribute ) {
+
+					positions = {
```

Comment:
Is there a way to avoid creating these objects? Maybe refactoring something from outside?